### ISSUE TITLE
400 error on DELETE request

### ISSUE THREAD

**Issue by wwwroth** [ixudra/curl#26]
I'm running a Laravel project within a Vagrant VM and am attempting to use `delete()` at an API endpoint I've created. I kept getting false, so I used `enableDebug()` and uncovered I was getting a 400 error. GET, POST, PUT, PATCH work well. Here's my log. 

```
* Hostname was found in DNS cache
* Hostname in DNS cache was stale, zapped
*   Trying 127.0.53.53...
* Connected to ***-api.***.dev (127.0.53.53) port 80 (#74)
> DELETE /accounts/59/payment-methods/delete/card_**** HTTP/1.1
Host: ***-api.***.dev
Accept: */*

* The requested URL returned error: 400 Bad Request
* Closing connection 74
```
I removed where I was doing it with your lib and tried the request with PHP's standard CURL lib and it worked fine.
```
//$response = Curl::to($endpoint)->delete();
$ch = curl_init();
curl_setopt($ch, CURLOPT_URL, $endpoint);
curl_setopt($ch, CURLOPT_CUSTOMREQUEST, "DELETE");
curl_setopt($ch, CURLOPT_POSTFIELDS, json_encode($data));
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
$result = curl_exec($ch);
$result = json_decode($result);
curl_close($ch);

return json_encode($result);
```
I'll be looking into the service and trying to see if I can patch it. If I can, I'll submit a PR.

**Comment by elimentz** [comment]
Can you include the request that you made with the package? Is there some way I can replicate it or is it a custom API?

**Comment by wwwroth** [comment]
It's a custom, private API I have running locally unfortunately. I've been going through Builder.php and it seems that you're setting all the same curl options identical to how I did it through PHP's lib. Still debugging, though.

**Comment by elimentz** [comment]
Fixed in [#27](https://github.com/ixudra/curl/pull/27)
